packages/clerk-js/src/utils/web3.ts

```diff
@@ -90,6 +98,17 @@ async function getEthereumProvider(provider: Web3Provider) {
     });
     return sdk.getProvider();
   }
+  if (provider === 'base_wallet') {
+    try {
+      const { createBaseAccountSDK } = await import('@base-org/account');
+      const sdk = createBaseAccountSDK({
+        appName: typeof document !== 'undefined' ? document.title || 'Clerk' : 'Clerk',
+      });
+      return sdk.getProvider();
```

Comment:
🙃 Just make sure that this doesn't wind up in the No RHC/Chrome Extension build with `__BUILD_DISABLE_RHC__`. Some prior art is on line 86 of this file.

Reply:
[`dd94966` (#6556)](https://github.com/clerk/javascript/pull/6556/commits/dd9496630c90efc51523fb2b01c71489236ff890)